petisco/events/request_responded.py

```diff
@@ -25,11 +25,11 @@ def is_flask_response(value):
     return False
 
 
-def get_content(value):
-    if is_flask_response(value):
+def get_content(response, status_code):
+    if is_flask_response(response):
         return {"message": "flask response"}
     else:
-        return value
+        return {"message": "Response OK"} if status_code == 200 else response
```

Comment:
It should be valid for all 2XX status code.

Reply:
I'm not sure, the code 206 ("Scan needs more images") is threated as an error

Reply:
You are right